### ISSUE TITLE
Can't install screen

### ISSUE THREAD

**Issue by weirdcrap** [ich777/un-get#1]
Running v0.11

Testing out un-get and was able to install sshfs no problem, however screen seems to be broken?

`Something went wrong, package screen-4.9.0-x86_64-1.txz not found`

Steps to reproduce:

- un-get update
- un-get install screen
- Yes to run
- fails with error.
- Installs libkscreen instead of the requested screen package.

Pastebin of the terminal output: https://pastebin.com/DR7tQCTR

I was poking around in the plugin folder and looked at installed packages and it lists a package that's named similarly but is not screen.

![image](https://user-images.githubusercontent.com/22824785/186997934-cb4e5b60-dfee-4368-a006-ec136fc49363.png)

I tried to install by specific package name "screen-4.9.0" but it wouldn't work. 

EDIT: So it actually appears to have installed libkscreen-5.23.5-x86_64-1.txz in the /boot/extra folder despite it not being explicitly requested. 

Am I doing something wrong in how I'm requesting to install the package? I thought the explicit package name (screen-4.9.0) would work but it doesn't.

**Comment by ich777** [comment]
@weirdcrap thank you for the report.

Please update your plugin to version 2022.08.27 (v0.12), this should be fixed by now.

If you have no other packages in the extra directory which are installed through un-get then please run:
`un-get cleanup` <- this will cleanup all packages in /boot/extra and the installed package list for un-get
`un-get install screen`

> I tried to install by specific package name "screen-4.9.0" but it wouldn't work.

un-get was not designed to support specific version, if it finds two packages with the exact same name but a difference in version then it will always install the latest version.

**Comment by weirdcrap** [comment]
Updated version worked great. Thanks for the quick fix. I'm hoping they'll just include screen in the base OS one day. It's an indispensable tool for command line data transfers.

**Comment by ich777** [comment]
@weirdcrap if you are experiencing any other issues please let me know since I can't test everything and every use case... :laughing: 

If you have Unassigned Devices Plus installed maybe keep tmux a chance since it's integraded there.

Don't get me wrong but Unraid isn't a general purpose server so I really recommend to do everything through Docker or VMs, this tool was only created out of curiosity and because I wanted to know if I can create a thing that is not that dependent on the developer and would just "work".

**Comment by weirdcrap** [comment]
> @weirdcrap if you are experiencing any other issues please let me know since I can't test everything and every use case... 😆
> 
> If you have Unassigned Devices Plus installed maybe keep tmux a chance since it's integraded there.
> 
> Don't get me wrong but Unraid isn't a general purpose server so I really recommend to do everything through Docker or VMs, this tool was only created out of curiosity and because I wanted to know if I can create a thing that is not that dependent on the developer and would just "work".

I'll admit I've been to lazy to learn tmux as I'm already familiar with screen and it does everything I need it to. I'll give tmux a try though it would be nice to have one less extra package to worry about.

Oh I know, I was one of the users making noise over in the NerdPack thread on the forum when squid and dlandon said this lol. I'm only using nerdpack for two things, screen and sshfs-fuse both of which I feel squarely fall in the realm of server tools. 

If Unassigned Devices added the ability to do SSHFS mounts I wouldn't need any extra packages.

**Comment by Freddy-0** [comment]
I noticed a similar problem when trying to install gstreamer

```bash
root@Tower:~# un-get search gstreamer
Found the following package(s) for your search 'gstreamer' in repositories:
gstreamermm-1.10.0
gstreamer-1.18.5
phonon-backend-gstreamer-4.10.0
root@Tower:~# un-get install gstreamer
The following package(s) will be installed: gstreamer
Are you sure? [y/N] y

     0K ..                                                   100% 3.90M=0.03s

Package  already installed, skipping!
root@Tower:~# gst-launch-1.0 --version
-bash: gst-launch-1.0: command not found
root@Tower:~# grep "gstreamer" /boot/config/plugins/un-get/installedpackages_list
phonon-backend-gstreamer-4.10.0-x86_64-3.txz

```

Is there an easy way to specify only exact matches, or are you able to patch this behaviour?

**Comment by ich777** [comment]
@Freddy-0 I will look i to this ASAP.
You can‘t install a specific version, un-get will always grab the newest one.

Is this a package which is available in the official repository or a third party, if it‘s a third party repository please provide the repository link.

**Comment by Freddy-0** [comment]
gstreamer comes from the official sources. I did not install any third party sources. The version does not really matter. I wanted to install gstreamer and got phonon-backend-gstreamer instead.

**Comment by ich777** [comment]
@Freddy-0 the issue should be fixed with `un-get` Version `0.26`

Please open up a separate issue next time, I almost forgot about that.

**Comment by Freddy-0** [comment]
@ich777 Many Thanks. Issue is fixed with the newest Version. (Unraid - un-get 2024.04.09)
